backend-rust/src/transaction_event/baker.rs

```diff
@@ -129,6 +148,16 @@ pub struct BakerResumed {
     pub account_address: AccountAddress,
 }
 
+async fn account_address(baker_id: &BakerId, pool: &PgPool) -> ApiResult<AccountAddress> {
+    let address = sqlx::query_scalar!("SELECT address FROM accounts WHERE index = $1", baker_id.0)
+        .fetch_one(pool)
+        .await
+        .map_err(|_| {
+            ApiError::InternalError(format!("Unable to find account with index {}", baker_id))
+        })?;
+    Ok(AccountAddress::from(address))
+}
```

Comment:
Propagating the context to avoid repeating the line `let pool = get_pool(ctx)?;`.

```suggestion
async fn account_address(baker_id: &BakerId, ctx: &Context<'_>) -> ApiResult<AccountAddress> {
    let pool = get_pool(ctx)?;
    
    let address = sqlx::query_scalar!("SELECT address FROM accounts WHERE index = $1", baker_id.0)
        .fetch_one(pool)
        .await
        .map_err(|_| {
            ApiError::InternalError(format!("Unable to find account with index {}", baker_id))
        })?;
    Ok(AccountAddress::from(address))
}
```